sql/4_1_2-to-4_1_3_upgrade.sql

```diff
@@ -2782,3 +2782,10 @@ INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_re
 INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_release_date`, `load_filename`, `load_checksum`) VALUES ('ICD10', 'CMS', '2014-10-01', 'Reimbursement_Mapping_pr_2015.zip', '493c022db17a70fcdcbb41bf0ad61a47');
 #EndIf
 
+#IfTable drugs
```

Comment:
Maybe change to 
#IfNotColumnType drugs size varchar
ALTER TABLE `drugs` CHANGE `size` `size` varchar(25)  default NULL ;
#EndIf

This will allow slightly longer drug combinations. I know in Aus we have a 3 molecule combination. eg. 30/12.5/325